index.html

```diff
@@ -1,29 +1,163 @@
+                </ul>
+                <button class="btn-Project" type="submit">See Project</button>
+
+            </section> 
+            <section class="works">
+                <div>
+                    <img class="snapshoot" src="images/Snapshoot-Portfolio4.svg" alt="Snapshoot-Image">
+                </div>
+                <h2 class="tonic">Multi-Post Stories</h2>
+                <ul class="sub-Points">
+                    <li class="canopy">CANOPY</li>
+                    <li class="sub-Text">Back End Dev</li>
+                    <li class="sub-Text">2015</li>
+                </ul>
+                <p class="description-Text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li><button class="languages-List" type="submit">html</button></li>
+                   <li><button class="languages-List" type="submit">css</button></li>
+                    <li><button class="languages-List" type="submit">javascript</button></li>
+                </ul>
+                <button class="btn-Project" type="submit">See Project</button>
+            </section>
+            <section class="about-Myself">
+                <h2 class="about-Text">About<br>Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <h2 class="connect">LET'S CONNECT</h2>
+                <ul>
+                    <li><img class="section1-Icons" src="images/Frame1.svg" alt="social-Media-Icons"></li>
+                </ul>
+                <button class="btn-Project2" type="submit">See Resume</button>
+                <p class="tools">Languages <img src="images/downicon.svg" alt="tools"> </p>
+                <ul>
+                    <li class="list-Class"><img src="images/Ellipsej.svg" alt=""><div class="ellipse-J">JavaScript</div></li>
+                    <li class="list-Class"><img src="images/Ellipseh.svg" alt=""><div class="ellipse-H">HTML</div></li>
+                    <li class="list-Class"><img src="images/Ellipsec.svg" alt=""><div class="ellipse-C">CSS</div></li>
+                </ul>
+                <ul>
+                <li><p class="tools">Frameworks <img src="images/Unionright.svg" alt="tools"></p></li>
+                <li><p><img class="seperator" src="images/SeparatorBottom.svg" alt="seperator"></p></li>
+                <li><p class="tools">Skills <img src="images/Unionright.svg" alt="tools"></p></li>
+                <li><p><img class="seperator" src="images/SeparatorBottom.svg" alt="seperator"></p></li>
+            </ul>
+            <br>
+                <br>
+                <br>
+                <br>
+                <br>
+                <br>
+                <br>
```

Comment:
[OPTIONAL] I strongly suggest you use `CSS` properties to handle the space between the `sections` you have great options like `padding` or `margin` for this purpose.

Reply:
completed